src/components/ReactTags.tsx

```diff
@@ -70,8 +79,9 @@ const ReactTags = (props: ReactTagsProps) => {
     maxLength,
     inputValue,
     clearAll,
+    enableFuzzySearch = false,
   } = props;
-
+  const maximumFuzzyDistance = 5;
```

Comment:
@ad1992 is it ok to keep it here or shall I move it outside the component?

Reply:
This isn't needed anymore right ? so lets remove it

Reply:
removed https://github.com/react-tags/react-tags/pull/985/commits/0e4de76a8ae12e8b3b9f676ab80f74116eb53ee1